src/client/Microsoft.Identity.Client/Platforms/iOS/iOSPlatformProxy.cs

```diff
@@ -98,6 +99,11 @@ public override ILegacyCachePersistence CreateLegacyCachePersistence()
             return new iOSTokenCacheAccessor();
         }
 
+        public override string GetEnrollmentIds()
+        {
+            return NSUserDefaults.StandardUserDefaults.StringForKey(EnrollmentIdKey);
```

Comment:
Does this return null or throw if the key doesn't exist?

Reply:
returns null